hack/make/.integration-test-helpers

```diff
@@ -52,6 +52,7 @@ test_env() {
 		DOCKER_HOST="$DOCKER_HOST" \
 		DOCKER_REMAP_ROOT="$DOCKER_REMAP_ROOT" \
 		DOCKER_REMOTE_DAEMON="$DOCKER_REMOTE_DAEMON" \
+		DOCKERFILE="$DOCKERFILE" \
 		GOPATH="$GOPATH" \
```

Comment:
for the multi-arch failures,if you add in TEST_IMAGE_NAMESPACE here its recognized it later on and all the images are tagged correctly. However there appears to be an issue with deleteAllImages() removing the images after the first test and not getting them back again as a lot of the tests fail. (I'm not 100% sure how this works currently)